nifi-commons/nifi-web-client-api/src/main/java/org/apache/nifi/web/client/api/HttpResponseStatus.java

```diff
@@ -55,4 +57,12 @@ public enum HttpResponseStatus {
     public int getCode() {
         return code;
     }
+
+    public boolean isSuccessful() {
+        return isSuccessful(code);
+    }
+
+    public static boolean isSuccessful(final int code) {
+        return code >= 200 && code < 300;
+    }
```

Comment:
I will probably just remove both then since the member method isn't really helping since there is no way to get an instance of `HttpResonseStatus` from a code.